cfgov/cfgov/settings/base.py

```diff
@@ -767,7 +767,9 @@
     # Hide archive filter options in the filterable UI
     "HIDE_ARCHIVE_FILTER_OPTIONS": [],
     # Enable ES as the backend for FilterableLists
-    "ELASTICSEARCH_FILTERABLE_LISTS": []
+    "ELASTICSEARCH_FILTERABLE_LISTS": [],
+    # Expand ES Filterable List Search
+    "EXPAND_FILTERABLE_LIST_SEARCH": []
```

Comment:
Incredibly minor thing, can we add a trailing `,` here?
```suggestion
    "EXPAND_FILTERABLE_LIST_SEARCH": [],
```